src/components/errorbars/plot.js

```diff
@@ -117,6 +118,7 @@ module.exports = function plot(traces, plotinfo, transitionOpts) {
 
                 if(isNew) {
                     xerror = errorbar.append('path')
+                        .style('vector-effect', 'non-scaling-stroke')
```

Comment:
right - because that's about the *length* of the whisker, not its stroke. I couldn't think of a way to handle this at first but now that you mention it, I suppose we could have the `x` scale of the `y` errorbars (and the `y` scale of `x` error bars) get inverse-scaled the same way as we do for both x and y scales of the points... I can look into this but it may take a little more work, do you think it's worth holding up this PR or shall I tackle it separately later?